doc/r_c_shortest_paths.html

```diff
@@ -329,6 +329,11 @@ <h4>DominanceFunction</h4>
 <table border>
 <tr>
 <td>
+<tt>vis.on_enter_loop(unprocessed_labels, const Graph&amp; g )</tt>
```

Comment:
Shouldn't it rather be this?
```suggestion
<tt>vis.on_enter_loop(const Queue&amp; queue, const Graph&amp; g )</tt>
```
It looks like you accidentally mixed arguments and parameters together, but I could be wrong.

Reply:
You are right @jeremy-murphy , I was unsure about the convention in the docs and which type to use.